docs/courses/csintro/blocks/unit-2/lab0201-part1.md

```diff
@@ -1,13 +1,13 @@
-# Lab 2.1 Part 1: Shooting gallery
+# Lab 2.1 Part 1: Aiming Gallery
 
-## Fire away! @showdialog
+## Fire Away! @showdialog
 
 In this lab, you will learn about event handlers and how they can be used
 in your projects.
 
 You also will learn ways to play sounds.
 
-## Step right up!
+## Step Right Up!
 
 We will be creating a "shooting gallery" game.
```

Comment:
Do we have to update "shooting gallery" everywhere or just in the main title?

Reply:
I'll probably end up doing it everywhere.